resilience4j/tests/test_resilience4j.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from datadog_checks.dev.utils import get_metadata_metrics
+from datadog_checks.resilience4j import Resilience4jCheck
+
+EXPECTED_PROMETHEUS_METRICS = {
+    "resilience4j.bulkhead.available.concurrent.calls",
+    "resilience4j.bulkhead.max.allowed.concurrent.calls",
+    "resilience4j.bulkhead.max.thread.pool.size",
+    "resilience4j.bulkhead.queue.capacity",
+    "resilience4j.bulkhead.queue.depth",
+    "resilience4j.bulkhead.thread.pool.size",
+    "resilience4j.circuitbreaker.buffered.calls",
+    "resilience4j.circuitbreaker.calls.seconds.count",
+    "resilience4j.circuitbreaker.calls.seconds.max",
+    "resilience4j.circuitbreaker.calls.seconds.sum",
+    "resilience4j.circuitbreaker.calls",
+    "resilience4j.circuitbreaker.failure.rate",
+    "resilience4j.circuitbreaker.max.buffered.calls",
+    "resilience4j.circuitbreaker.state",
+    "resilience4j.ratelimiter.available.permissions",
+    "resilience4j.ratelimiter.waiting.threads",
+    "resilience4j.retry.calls",
+    "resilience4j.timelimiter.calls",
```

Comment:
```suggestion
    "bulkhead.available.concurrent.calls",
    "bulkhead.max.allowed.concurrent.calls",
    "bulkhead.max.thread.pool.size",
    "bulkhead.queue.capacity",
    "bulkhead.queue.depth",
    "bulkhead.thread.pool.size",
    "circuitbreaker.buffered.calls",
    "circuitbreaker.calls.seconds.count",
    "circuitbreaker.calls.seconds.max",
    "circuitbreaker.calls.seconds.sum",
    "circuitbreaker.calls",
    "circuitbreaker.failure.rate",
    "circuitbreaker.max.buffered.calls",
    "circuitbreaker.state",
    "ratelimiter.available.permissions",
    "ratelimiter.waiting.threads",
    "retry.calls",
    "timelimiter.calls",
```
Adding the `__NAMESPACE__` will automatically assert the integration name prefix in tests.